streaming-connect-sink/src/main/java/com/adobe/platform/streaming/sink/AbstractAEPPublisher.java

```diff
@@ -57,6 +57,8 @@
 
   private static final String AEP_CONNECTION_AUTH_ENABLED_VALUE = "true";
   private static final String AEP_CONNECTION_AUTH_DISABLED_VALUE = "false";
+  public static final String AEP_ERROR_LOGGER = "aep.error.logger";
+  public static final String AEP_ERROR_TOPIC = "aep.error.topic";
```

Comment:
Agreed with @vivetiwa that we did not settle for the basic Kafka connect DLQs because of the behaviour of stopping the entire stream when there is a error encountered.